frontend/js/components/JoinParty/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import '../css/MainScreen.css';
+import { Button } from 'reactstrap';
+
+
+const JoinParty = () => {
```

Comment:
Probably rename this to and put inside a folder like `JoinPartyList.js` and then have `components/JoinParty/index.js` look like:

```
export { default as JoinPartyList } from './JoinPartyList';
export { default as JoinPartyName } from './JoinPartyName';
```

Having `index` be dedicated to a single component when there's multiple components in the folder can get confusing.